projects/packages/forms/src/blocks/contact-form/components/jetpack-integrations-modal/integrations-list.tsx

```diff
@@ -41,18 +42,21 @@ const IntegrationsList = ( {
 	const [ expandedCards, setExpandedCards ] =
 		useState< ExpandedCardsState >( initialCardsExpandedState );
 
-	const toggleCard = useCallback( ( id: string ) => {
-		setExpandedCards( prev => {
-			const isExpanding = ! prev[ id ];
-			if ( isExpanding ) {
-				jetpackAnalytics.tracks.recordEvent( 'jetpack_forms_integrations_card_expand', {
-					card: id,
-					origin: 'block-editor',
-				} );
-			}
-			return { ...prev, [ id ]: isExpanding };
-		} );
-	}, [] );
+	const toggleCard = useCallback(
+		( id: string ) => {
+			setExpandedCards( prev => {
+				const isExpanding = ! prev[ id ];
+				if ( isExpanding ) {
+					jetpackAnalytics.tracks.recordEvent( 'jetpack_forms_integrations_card_expand', {
+						card: id,
+						origin: context,
+					} );
+				}
+				return { ...prev, [ id ]: isExpanding };
+			} );
+		},
+		[ context ]
+	);
```

Comment:
This part of diff is cleaner if you hide whitespace. We're just updating this to pass context for tracking, ie, `origin: context`